bin/compile

```diff
@@ -318,6 +345,8 @@ build_dependencies() {
     yarn_2_install "$BUILD_DIR"
   elif $YARN; then
     yarn_node_modules "$BUILD_DIR"
+  elif $PNPM; then
```

Comment:
Does this mean we always have to set a `PNPM` env var on our Heroku apps? (apparently not looking at the Heroku review apps, so clearly I've misunderstood something! - guessing it's somehow set if a pnpm lockfile is found, as per the comment on L240?)

Reply:
Ah, missed that - thx